CMakeLists.txt

```diff
@@ -650,6 +650,23 @@ set(
 )
 mark_as_advanced(HELICS_CMAKECONFIG_INSTALL_DIR)
 
+set(HELICS_INSTALL_TARGETS helics_base helics_base_includes jsoncpp_lib helics-static)
+if(BUILD_SHARED_LIBS)
+    set(HELICS_INSTALL_TARGETS ${HELICS_INSTALL_TARGETS} helics-shared)
+endif()
+if(BUILD_C_SHARED_LIB)
+    set(HELICS_INSTALL_TARGETS ${HELICS_INSTALL_TARGETS} helicsSharedLib helicsCpp98)
+endif()
+if(BUILD_APPS)
+    set(HELICS_INSTALL_TARGETS ${HELICS_INSTALL_TARGETS} helics_apps)
+endif()
+file(GLOB HELICS_CMAKE_MODULES "config/cmake/*.cmake")
```

Comment:
Do we really need to copy all of these over?

Reply:
I think only the FindZeroMQ and addMPI files are needed.

Reply:
Should we just copy those files vs all of them?

Reply:
This is updated now to only copy those two files. If we ever bump the required CMake version to 3.9+ then we can get rid of addMPI.cmake